src/gui/creds/webflowcredentials.cpp

```diff
@@ -140,32 +139,17 @@ void WebFlowCredentials::fetchFromKeychain() {
 }
 
 void WebFlowCredentials::askFromUser() {
-    // Determine if the old flow has to be used (GS for now)
-    // Do a DetermineAuthTypeJob to make sure that the server is still using Flow2
-    auto job = new DetermineAuthTypeJob(_account->sharedFromThis(), this);
-    connect(job, &DetermineAuthTypeJob::authType, [this](DetermineAuthTypeJob::AuthType type) {
-        // LoginFlowV2 > WebViewFlow > OAuth > Shib > Basic
-        bool useFlow2 = (type != DetermineAuthTypeJob::WebViewFlow);
-
-        _askDialog = new WebFlowCredentialsDialog(_account, useFlow2);
-
-        if (!useFlow2) {
-            QUrl url = _account->url();
-            QString path = url.path() + "/index.php/login/flow";
-            url.setPath(path);
-            _askDialog->setUrl(url);
-        }
```

Comment:
Why did you remove the auth job? I think you shouldn't remove it.

Reply:
As far as I can tell it was dead code since it's only needed to distinguish whether flowv1 is needed over v2. Do you want an error message or something in that case instead?

Reply:
For me, it looks like it also checked if basic authentication is needed.